components/Header/src/index.scss

```diff
@@ -0,0 +1,10 @@
+.denhaag-header {
+  border-bottom: var(--denhaag-header-border-bottom);
```

Comment:
If we use `--denhaag-header-border-bottom-color`, `--denhaag-header-border-bottom-width` etc. and a hardcoded `border-bottom-style: solid;` we will be able to use the same key/value pairs in Figma as this doesn't support the same composite tokens. The same goes for a hypothetical other output such as for iOS or Android. :)